src/test/java/org/broadinstitute/hellbender/tools/spark/transforms/BaseRecalibratorSparkFnUnitTest.java

```diff
@@ -0,0 +1,81 @@
+package org.broadinstitute.hellbender.tools.spark.transforms;
+
+import com.google.cloud.genomics.dataflow.utils.GCSOptions;
+import htsjdk.samtools.SAMFileHeader;
+import org.apache.spark.api.java.JavaPairRDD;
+import org.apache.spark.api.java.JavaRDD;
+import org.apache.spark.api.java.JavaSparkContext;
+import org.broadinstitute.hellbender.engine.dataflow.datasources.ReadContextData;
+import org.broadinstitute.hellbender.engine.dataflow.datasources.RefAPISource;
+import org.broadinstitute.hellbender.engine.dataflow.datasources.RefWindowFunctions;
+import org.broadinstitute.hellbender.engine.dataflow.datasources.ReferenceDataflowSource;
+import org.broadinstitute.hellbender.engine.spark.AddContextDataToReadSpark;
+import org.broadinstitute.hellbender.engine.spark.JoinReadsWithVariants;
+import org.broadinstitute.hellbender.engine.spark.SparkContextFactory;
+import org.broadinstitute.hellbender.engine.spark.datasources.ReadsSparkSource;
+import org.broadinstitute.hellbender.engine.spark.datasources.VariantsSparkSource;
+import org.broadinstitute.hellbender.tools.IntegrationTestSpec;
+import org.broadinstitute.hellbender.tools.dataflow.pipelines.BaseRecalibratorDataflow;
+import org.broadinstitute.hellbender.utils.recalibration.RecalibrationArgumentCollection;
+import org.broadinstitute.hellbender.utils.recalibration.RecalibrationReport;
+import org.broadinstitute.hellbender.utils.IntervalUtils;
+import org.broadinstitute.hellbender.utils.SimpleInterval;
+import org.broadinstitute.hellbender.utils.read.GATKRead;
+import org.broadinstitute.hellbender.utils.spark.JoinStrategy;
+import org.broadinstitute.hellbender.utils.test.BaseTest;
+import org.broadinstitute.hellbender.utils.variant.Variant;
+import org.testng.annotations.DataProvider;
+import org.testng.annotations.Test;
+
+import java.io.File;
+import java.io.PrintStream;
+import java.util.List;
+
+public class BaseRecalibratorSparkFnUnitTest extends BaseTest {
```

Comment:
This appears to be an early draft version of `BaseRecalibratorSparkIntegrationTest` left over from the rebase (notice how most of the test cases are commented out!). Should be deleted.